modelexpress_client/python/tests/test_refit_trainer_s3.py

```
@@ -0,0 +1,729 @@
+        assert staged._staged.encoded_deltas == {}
+        assert staged._staged.checksums == {}
+    finally:
+        trainer.close()
+
+    assert service.shards == []
+
+
+def test_s3_processes_buckets_concurrently_and_uploads_one_shard(
+    monkeypatch, tmp_path, refit_server
+):
+    service, server_url = refit_server
```

Ruff (0.16.2)

[warning] Unpacked variable `service` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)